### ISSUE TITLE
Trouble instantiating a custom store with v5 upgrade: solution

### ISSUE THREAD

**Issue by EdmundsEcho** [jakearchibald/idb-keyval#118]
Great little resource.  

I've used it for a couple of years now both to persist my redux state and provide a react-hook service to store local ui state. 

Recently, I went to clean out my browser memory... I then ran into instantiation "Unhandled... can't find IDB ... error".

The documentation you provide here, where `Store` no longer works, was spot-on.  

I'm only writing this issue to augment what is now obvious.  The inability to create more than one table for a given database seemed to prevent my getting things back in order i.e., it was blocking Firefox from "resetting" or otherwise reading from the previous store, the store instantiated using v3.

In other words, the docs were clear but where it played out was surprising because I was not trying to use separate tables in the same db. So I did not think it related to my issue.

What I needed to do was rename both the database and table, for where I use it in the app.  For instance, just renaming the database did not seem to "do the trick".  Once I renamed both, both Firefox and Vivaldi created the new stores using v5.

Thank you.  I hope this helps.

**Comment by jakearchibald** [comment]
Thanks for the feedback! Yeah, I tried adding code to allow for multiple tables per database, but it ended up being more complicated than just using https://github.com/jakearchibald/idb/.
